swagger.json

```diff
@@ -1121,6 +1121,122 @@
             }
           }
         },
+        "/fleet/drivers/{driver_id}/hos_certified_day_chart/{time_ms}": {
```

Comment:
Do we want to expose this as a day chart? This is going to open up a can of worms for noon to noon fleets :(

Reply:
I wrote it since I saw the term `day chart` being used in the legal lingo, but I'm happy to just keep it to `hos_certified_chart` if that sounds reasonable

Reply:
Is this really about certifying the driver's `shift`? Would `hos_certified_shift` make sense? 

In either case, I think we should document what value they need to pass for timeMs, and how we resolve that timestamp to their shift logs.

Reply:
Does the description I provided for `time_ms` parameter work to clarify that the timestamp should be in the bounds of a driver's shift? Or should I provide an extra description at the getter and patch level?